.props/Product.props

```diff
@@ -9,7 +9,7 @@
   <Import Project=".\_AnalyzerSettings.props" />
 
   <Import Project=".\_GlobalStaticVersion.props" />
-  <Import Project=".\_Nupkg.props" />
+  <Import Project=".\_Nupkg.props"  Condition="'$(IsExamplesSolution)' != 'true'"/>
```

Comment:
Excluding the Nupkg.props in the Examples solution will block our SDKs from building nuget packages. this will speed  up compilation when working with Examples